### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
- Image related CSS you can find in Image.css file
- input is added using form releated css you can find in input.css
- form validation is added related js written in component.js file.

**Screenshots**

respimage
roundimg
input

### CODE REVIEW

**Review by sakleprajwal — COMMENTED** [review]
There is a conflict showing in this PR. Other than that everything is good.

**Comment by kotalwar-krishna** [comment]
> There is a conflict showing in this PR. Other than that everything is good.
conflicts resolved

**Review by paarthbajaj — APPROVED** [review]
All looks good. Just want to know why you have used JS for validation?

File: components/component.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        result.style.color="red";
+        inputs[i].style.border="2px solid red";
+}
+
+const successMessage = (result) => {
+        result.childNodes[1].innerText=`check_circle`;
+        result.childNodes[3].innerText= `Looks Good`;
+        result.style.color="green";
+        inputs[i].style.border="2px solid green"; 
+}
```
Comment (vaibhav18matere) [inline components/component.js:45]: I think we have to do a Component library in pure HTML and CSS. No need to add JS. PLease confirm we can add it or not from captains

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.form-container{
+    padding: 1rem;
+    margin: 0.5rem;
+    font-size: 1rem;
+    background-color: lightgreen;
```
Comment (vaibhav18matere) [inline components/input.css:5]: use color variables instead of direct colors

**Review by vaibhav18matere — APPROVED** [review]
class names used are pretty informative, also good work in UI. check following small edits though!

File: components/component.js (hunk 1 above)
Comment (kotalwar-krishna) [reply components/component.js:45]: ok i will

File: components/input.css (hunk 2 above)
Comment (kotalwar-krishna) [reply components/input.css:5]: ya i think that after all the work of component library done. after that i will refactor

**Review by sakleprajwal — COMMENTED** [review]
Approved

**Pull request merged**